src/components/Lookup/index.js

```diff
@@ -668,8 +668,8 @@ Lookup.propTypes = {
     /** The variant changes the appearance of the Input. Accepted variants include default,
      * shaded and bare. This value defaults to default. */
     variant: PropTypes.oneOf(['default', 'shaded', 'bare']),
-    /** The border radius of the button. Valid values are square, semi-rounded and rounded. This value defaults to rounded. */
-    borderRadius: PropTypes.oneOf(['square', 'semi-rounded', 'rounded']),
+    /** The border radius of the input. Valid values are square, semi-square, semi-rounded and rounded. This value defaults to rounded. */
+    borderRadius: PropTypes.oneOf(['square', 'semi-square', 'semi-rounded', 'rounded']),
```

Comment:
Look here the inner border radius for semi square does not match the outside one

<img width="586" alt="Screen Shot 2022-11-27 at 10 24 00 AM" src="https://user-images.githubusercontent.com/24737097/204152863-eeb7a720-62a1-41e7-9ca9-4c192a4ba21f.png">